sdk/ai/Azure.AI.AgentServer/samples/AgentFramework/MinimalConsole/README.md

```diff
@@ -0,0 +1,19 @@
+# Agent Framework Minimal Console Sample
+This is a minimal console application that demonstrates how to use the Agent Framework library to create and run an agent with a simple task.
+
+## Prerequisites
+- [.NET 8.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) or later
+- Access to github packages for nightly builds, see [FAQs](../../README.md#faqs) below.
+
+## Getting Started
+```csharp
```

Comment:
All code in markdown files and XML doc comments in the repository must be in the form of [snippets](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets).

Also see [Azure SDK for .NET sample guideines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples) for information on the required front matter to use in the README.